w3af/core/ui/api/utils/cli.py

```diff
@@ -61,6 +61,12 @@ def parse_arguments():
                         default=False,
                         nargs='?')
 
+    parser.add_argument('--no-ssl',
+                        dest="disableSSL",
```

Comment:
Please check [these lines](https://github.com/andresriancho/w3af/blob/master/w3af/core/ui/api/utils/cli.py#L185-L187) , I believe the disable ssl setting should also show/hide that message.